### PR TITLE
For #6577 - `total_uri_count` Glean telemetry

### PR DESCRIPTION
**Pull request by Mugurell**
Use `total_uri_count` as a Glean CounterMetricType tracked in both the `baseline` and `metrics` ping.
This would also complement the changes for https://github.com/mozilla-mobile/fenix/issues/4456.

### Pull Request checklist

- [x] **Tests**: This PR does not include tests as it only refactor specific telemetry events.
- [x] **Screenshots**: This PR does not include screenshots as the changes are only for telemetry. 
- [x] **Accessibility**: The code in this PR does not include any user facing features.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by boek** [comment]
Please fill out a [data review request form](https://github.com/mozilla/data-review/blob/master/request.md) and add it as a comment here, I'll review it right away 😁!

**Comment by Mugurell** [comment]
# Request for data collection review form

**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**

1) What questions will you answer with this data?
A counter of URIs visited by the user in the current session, including page reloads.
This does not include background page requests and URIs from embedded pages or private browsing.

2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements? Some example responses:
This allows for a better assessment of the app's usage.

3) What alternative methods did you consider to answer these questions? Why were they not sufficient?
N/A

4) Can current instrumentation answer these questions?
This comes to improve the way we are collected data initially requested in https://github.com/mozilla-mobile/fenix/issues/1301

5) List all proposed measurements and indicate the category of data collection for each measurement, using the [Firefox data collection categories](https://wiki.mozilla.org/Firefox/Data_Collection) found on the Mozilla wiki.   

**Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.**

<table>
  <tr>
    <td>Measurement Description</td>
    <td>Data Collection Category</td>
    <td>Tracking Bug #</td>
  </tr>
  <tr>
    <td>uri visits</td>
    <td>Category 2</td>
    <td>https://github.com/mozilla-mobile/fenix/issues/1301
            https://github.com/mozilla-mobile/fenix/issues/4456
    </td>
  </tr>
</table>


6) How long will this data be collected?  Choose one of the following:
This is scoped to a time-limited experiment/project until date 09-01-2020.

7) What populations will you measure?
All release, beta, and nightly users with telemetry enabled.

8) If this data collection is default on, what is the opt-out mechanism for users?
Users can opt of of data collection by disabling `Usage and technical data` from Settings -> Data collection.

9) Please provide a general description of how you will analyze this data.
Glean / Amplitude

10) Where do you intend to share the results of your analysis?
Only on Glean, Amplitude and with mobile teams.

11) Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?
No.

File: app/src/migration/java/org/mozilla/fenix/MigrationService.kt [hunk 1]
```diff
@@ -7,11 +7,13 @@ package org.mozilla.fenix
...
 import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.migration.MigrationProgressActivity
 
 /**
  * Background service for running the migration from legacy Firefox for Android (Fennec).
  */
 class MigrationService : AbstractMigrationService() {
     override val migrator by lazy { getMigratorFromApplication() }
     override val store: MigrationStore by lazy { components.migrationStore }
+    override val migrationActivity = MigrationProgressActivity::class.java
```
[the remark this answers is not in the document]
Comment (Mugurell) [reply app/src/migration/java/org/mozilla/fenix/MigrationService.kt]: Definitely. Thanks!

**Comment by Dexterp37** [comment]
Oh, let's also wait for @sblatz to review this, since I don't own this repo :-P

**Pull request merged**